corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -81,10 +82,23 @@ public void listTables(String namespace)
     {
         log.info("\n=====Tables=======\n");
         for (CorfuStoreMetadata.TableName tableName : runtime.getTableRegistry()
-            .listTables(namespace)) {
+                .listTables(namespace)) {
             log.info("Table: " + tableName.getTableName());
             log.info("Namespace: " + tableName.getNamespace());
         }
         log.info("\n======================\n");
     }
+
+    /**
+     * Print information about a specific table in CorfuStore
+     */
+    public void printTableInfo(String namespace, String tablename) {
+        log.info("\n======================\n");
+        String fullName = TableRegistry.getFullyQualifiedTableName(namespace, tablename);
+        UUID streamUUID = UUID.nameUUIDFromBytes(fullName.getBytes());
+        CorfuTable<CorfuDynamicKey, CorfuDynamicRecord> table = getTable(namespace, tablename);
+        log.info("Table " + tablename + " in namespace " + namespace
```

Comment:
Use {} as a placeholder to make this statement clearer.

Reply:
done. thanks